components/cards/card.css

```diff
@@ -0,0 +1,241 @@
+
+@import "/reset.css";
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+
+}
+html{
+    font-size: 72%;
+}
+
+.hero{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 39rem;
+    width: 32rem;
+    margin: 4rem;
+    border: var(--card-text);
+    border-radius: 8px;
+    background-color: var(--card-bg-color);
+    overflow: hidden;
+    box-shadow: 2px 2px 6px 3px darkgrey;
+}
+
+img{
```

Comment:
class name and properties do not match.
give a meaningful name.

Reply:
ohk ig its for image height and width ok i change it